.github/workflows/hello-reason.yml

```
@@ -33,18 +37,17 @@ jobs:
             "source": "./package.json",
             "override": {
               "dependencies": {
-                "reason-mobile": "github:EduardoRFS/reason-mobile:generate.json#'"${GITHUB_SHA}"'"
+                "${{ github.event.repository.name }}": "github:'"${GITHUB_REPOSITORY}"':generate.json#'"${GITHUB_SHA}"'"
```

```suggestion
                "reason-mobile": "github:'"${GITHUB_REPOSITORY}"':generate.json#'"${GITHUB_SHA}"'"
```
The name of the package doesn't matter, but I'm not sure all github repository name will be a valid esy repository